### DISCUSSION TITLE
No communication with

### DISCUSSION THREAD

**Discussion by Glurik** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#719] [Q&A]
Hi folks.
I have MAIAMI Model S100102.  Main Board version:
![Main Board](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/91469119/8941df4f-c70f-4358-b37e-c71848a5d36c)
Display version:
![Display](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/91469119/ccc31aa6-83cc-4ec3-aba8-d51dd2560052)
I'd make D1-mini board by Raomin instructions, the web UI and MQTT works good.
hwtest passed without errors. Display shows IP and date on boot. All buttons presses sounds, Power button and F led lights permanently and digits display not lightning. Presses on any button do nothing, so via web UI.

My Hardware config in web UI:
CIO -  6 wire, 54149E (air)
DSP - 6 wire, pre 2021 (air)
PCB: Custom
Pinout: CIO - 4,3,2 - DSP 7,6,5,0
I tried all possible variations of the config, this is the best result I'd get. Others or nothing work or without sounds on buttons presses.
It's looks like the main board have different set of commands. 

Any idea , what wrong?

**Comment by metaljay** [comment]
So just to confirm, display works upon boot and then after booting it turns off?
Have you checked if display buttons are enabled in spa config

**Reply by Glurik** [reply]
No, display didn't turn off after boot. It lights only two leds permanently. each button have different sound on press. And display buttons are enabled in spa config, It's by default.
It's looks like interaction between display and D1-mini is Ok.

**Reply by metaljay** [reply]
Ah. This is why you need the low level converter, I also had no success with D1

**Reply by Glurik** [reply]
I'm not familar with LLC. Do you mean logic level converter? Like [this](https://www.aliexpress.com/item/1005003772569293.html?spm=a2g0o.detail.pcDetailTopMoreOtherSeller.6.7d9bBqtSBqtSBi&gps-id=pcDetailTopMoreOtherSeller&scm=1007.40050.354490.0&scm_id=1007.40050.354490.0&scm-url=1007.40050.354490.0&pvid=6e0c9f5b-53f8-41a1-a028-b5fe5a7c3ab7&_t=gps-id:pcDetailTopMoreOtherSeller,scm-url:1007.40050.354490.0,pvid:6e0c9f5b-53f8-41a1-a028-b5fe5a7c3ab7,tpp_buckets:668%232846%238111%231996&pdp_npi=4%40dis%21ILS%216.46%216.46%21%21%211.73%211.73%21%402140dcce17175139676863989ed8d4%2112000027112651637%21rec%21IL%21123417060%21&utparam-url=scene%3ApcDetailTopMoreOtherSeller%7Cquery_from%3A) ?
That means, the main board have to receive diffirent voltage? Which one, 3,3v or 5v ?

**Reply by metaljay** [reply]
This: https://a.aliexpress.com/_EuAEQf9

**Reply by Glurik** [reply]
Insert the LLC between D1 an CIO? Ther is schematics anywhere?

**Reply by metaljay** [reply]
Pass, I think that’s why most stick to the PCB board designed by VisualApproach

**Reply by Glurik** [reply]
Anyway, thank you very much for help.
